spec.bs

```diff
@@ -1272,6 +1281,9 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
      : [=automatic beacon data/destination=]
      :: |event|'s {{FenceEvent/destination}}
 
+     : [=automatic beacon data/once=]
+     :: |event|'s {{FenceEvent/once}}
```

Comment:
I'm assuming this comment is meant for the IDL definition, since I don't think there's anything that needs to be done here in `setReportEventDataForAutomaticBeacons()`.

Reply:
Ah yes sorry. I just happened to notice the issue _here_.